src/style.css

```
@@ -1 +1,7 @@
-/* styles go here */
+.groupDistance {
+  margin-bottom: 20px;/* styles go here */
+}
+
+.inputDistance {
+  margin-bottom: 10px;
+}/* styles go here */
```

```suggestion
.groupDistance {
  margin-bottom: 20px;
}

.inputDistance {
  margin-bottom: 10px;
  }
```
Do not leave comments